OSDOCS#7581,7582: Add token auth via CCO for Operator authors (AWS STS) (#64808)
openshift/openshift-docs: operators/operator_sdk/osdk-token-auth.adoc

:_content-type: ASSEMBLY
[id="osdk-token-auth"]
= Token authentication for Operators on cloud providers
include::_attributes/common-attributes.adoc[]
:context: osdk-token-auth

toc::[]

Many cloud providers can enable authentication by using account tokens that provide short-term, limited-privilege security credentials.

{product-title} includes the Cloud Credential Operator (CCO) to manage cloud provider credentials as custom resource definitions (CRDs). The CCO syncs on `CredentialsRequest` custom resources (CRs) to allow {product-title} components to request cloud provider credentials with any specific permissions required.

> Previously, on clusters where the CCO is in _manual mode_, Operators managed by Operator Lifecycle Manager (OLM) (for example, installed from the OperatorHub) often provided detailed instructions to users for how to manually provision any required cloud credentials.

Author: Reworked this to also address some ambiguity I wasn't feeling good about re: the "detailed instructions":
```suggestion
Previously, on clusters where the CCO is in _manual mode_, Operators managed by Operator Lifecycle Manager (OLM) often provided detailed instructions in the OperatorHub for how users could manually provision any required cloud credentials.
```
